package.json

```diff
@@ -139,5 +139,8 @@
         "types/*"
       ]
     }
+  },
+  "dependencies": {
+    "@ipld/dag-cbor": "^6.0.14"
```

Comment:
we'd best not add this in the top level or every user of `multiformats` will unnecessarily get it and we're trying to keep the package as minimal as possible

```suggestion
```